sp800-63b/sec10_usability.md

```diff
@@ -39,212 +36,218 @@ Positive user authentication experiences are integral to the success of an organ
 
 Usability considerations for intermittent events across authenticator types include:
-
 * To prevent users from needing to reauthenticate due to user inactivity, prompt users in order to trigger activity just before (e.g., 2 minutes before) an inactivity timeout would otherwise occur.
-*  Prompt users with adequate time (e.g., 1 hour) to save their work before the fixed periodic reauthentication event required regardless of user activity.
+* Prompt users with adequate time (e.g., 1 hour) to save their work before the fixed periodic reauthentication event required regardless of user activity.
 * Clearly communicate how and where to acquire technical assistance. For example, provide users with information such as a link to an online self-service feature, chat sessions or a phone number for help desk support. Ideally, sufficient information can be provided to enable users to recover from intermittent events on their own without outside intervention.
 
 ### 10.2. Usability Considerations by Authenticator Type
+In addition to the previously described general usability considerations applicable to most authenticators ([Section 10.1](#usabilitycommon)), the following sections describe other usability considerations specific to specific authenticator types.
 
-In addition to the previously described general usability considerations applicable to most authenticators (section 10.1.1), the following sections describe other usability considerations specific to specific authenticator types.
-
-
-#### 10.2.1. Memorized Secrets
-
+#### <a name="memorizedsecrets"></a> 10.2.1. Memorized Secrets
 **_Typical Usage_**
 
 Users manually input the memorized secret (commonly referred to as a password or PIN).
 
 Usability considerations for typical usage include:
 
-*  Memorability of the memorized secret:
-  *  The likelihood of recall failure increases as there are more items for users to remember; with fewer memorized secrets, users can more easily recall the specific memorized secret needed for a particular RP. The memory burden is greater for a less frequently used password.
+Memorability of the memorized secret.
 
-*  User experience during entry of the memorized secret:
-  *  Support copy and paste functionality in fields for entering memorized secrets, including passphrases.
+* The likelihood of recall failure increases as there are more items for users to remember; with fewer memorized secrets, users can more easily recall the specific memorized secret needed for a particular RP. The memory burden is greater for a less frequently used password.
+
+User experience during entry of the memorized secret.
+
+* Support copy and paste functionality in fields for entering memorized secrets, including passphrases.
 
 **_Intermittent Events_**
 
-Usability considerations for intermittent events include:
+Usability considerations for intermittent events include when users create and change memorized secrets:  
```

Comment:
This rewording doesn't make sense.  Maybe something like:

The use and changing of memorized secrets are intermittent events. Usability considerations for these events include: